### PR TITLE
Enable Mnemonics Keys for MacOS

### LINKED ISSUE #1726: Mac: Enable accelerator key functionality
Just found this in the [Qt docs](https://doc.qt.io/qt-5/qshortcut.html#details):

> On Mac, shortcuts are disabled by default. Call qt_set_sequence_auto_mnemonic() to enable them. However, because mnemonic shortcuts do not fit in with Aqua's guidelines, Qt will not show the shortcut character underlined.

We should probably consider activating them on Mac, too?
cc @softins @ann0see

_Originally posted by @hoffie in https://github.com/jamulussoftware/jamulus/discussions/1693#discussioncomment-755670_

### PR DESCRIPTION
**Pull request by henkdegroot**
Fixes #1726

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -39,13 +39,20 @@
...
 #endif
 
 // Implementation **************************************************************
 
 int main ( int argc, char** argv )
 {
 
+    // mnemonic keys are disabled by default on MacOS the following function enables them
+    // by "mac" design they will not be shown with underline characters in the GUI
+#if defined( Q_OS_MACX )
```
Comment (pljones) [inline src/main.cpp:52]: Could you move the comment inside the `#if`.

Could you also start new sentences in appropriate places to make the comment easier to read.

Comment (henkdegroot) [reply src/main.cpp:50]: Thanks for the comments, I have made the updates. Hope the text reads better now.

Comment (pljones) [reply src/main.cpp:50]: You've still not split it into sentences.  There are three separate ones but no full stops.

Comment (henkdegroot) [reply src/main.cpp:50]: For me there were just two sentences. I have updated the text.

Comment (pljones) [reply src/main.cpp:50]: > mnemonic keys are disabled by default on MacOS the following function enables them

This reads in multiple ways and hence needs punctuation

> mnemonic keys are disabled by default
> on MacOS the following function enables them

> mnemonic keys are disabled by default on MacOS
> the following function enables them

Also, as the line is fairly long, it's not clear whether the second line is part of the same sentance as the first due to the lack of punctuation.  Overall, it hampers reading and understanding.

Comment (henkdegroot) [reply src/main.cpp:50]: @pljones I have revised the text now. Are you checking the latest?

Comment (pljones) [reply src/main.cpp:50]: Apparently not...  Sorry!

**Review by pljones — APPROVED** [review]

**Review by ann0see — APPROVED** [review]
Tested client on Big Sur. Works as expected. --> merge

**Comment by ann0see** [comment]
CHANGELOG: Keyboard shortcuts now also work on macOS

**Comment by ann0see** [comment]
@hoffie I think this PR was wrongly added to the nightly changelog...

**Comment by hoffie** [comment]
Probably due to 0764536fab6b84c69fafb0b4a7e865cd088de346. I'll open a separate issue to avoid that.

**Pull request merged**